ResearchKit/Common/ORKPasscodeStepViewController.m

```diff
@@ -499,7 +499,7 @@ + (void)savePasscode:(NSString *)passcode withTouchIdEnabled:(BOOL)touchIdEnable
 
 - (void)removePasscodeFromKeychain {
     NSError *error;
-    [ORKKeychainWrapper objectForKey:PasscodeKey error:&error];
+    [ORKKeychainWrapper objectOfClass:NSDictionary.self forKey:PasscodeKey error:&error];
```

Comment:
This is technically unsafe. The contract for this kind of API is that if the result is nil, you can check whether there's more info in the error, but if the result is non-nil, there's no guarantee about what's in error. 
```suggestion
    id anyValue = [ORKKeychainWrapper objectOfClass:NSDictionary.self forKey:PasscodeKey error:&error];
    if (anyValue != nil) {
```